pkg/apis/hive/v1/hiveconfig_types.go

```diff
@@ -57,6 +57,13 @@ type HiveConfigStatus struct {
 	// configmap data from the openshift-config-managed namespace. When the configmap changes,
 	// admission is redeployed.
 	AggregatorClientCAHash string `json:"aggregatorClientCAHash,omitempty"`
+
+	// LastGenerationObserved will record the most recently processed HiveConfig object's generation.
+	LastGenerationObserved int64 `json:"lastGenerationObserved,omitempty"`
+
+	// ConfigApplied will be set by the hive operator to indicate whether or not the LastGenerationObserved
+	// was successfully reconciled.
+	ConfigApplied bool `json:"configApplied,omitempty"`
```

Comment:
It would be nice if this was a condition that stipulated why operator failed to apply the config. But that can be written up as a separate Jira card.